src/hooks/useKeyboardControls.ts

```diff
@@ -13,18 +13,17 @@ export const MovementKeys = {
   enter: ["enter"],
 };
 
-export const MOVEMENT_INTERVAL = 350;
-
 interface UseKeyboardControlsProps {
   venueId: string;
   totalRows: number;
   totalColumns: number;
   isSeatTaken: (row: number, column: number) => boolean;
   takeSeat: (row: number | null, column: number | null) => void;
   enterSelectedRoom: () => void;
-  onMove?: () => void;
 }
 
+// TODO: use e.preventDefault() or return false or similar in the keyboard handlers (check mousetrap docs) so we don't scroll on arrow key presses
+// TODO: we may also need some kind of 'centre me on the map' logic somewhere when we do this too? For large zoomed maps.
```

Comment:
I was also just thinking if you're within X% of the edge of the screen, and it has 'extra map' in that direction, then it will scroll 1 bubble at a time in that direction. Sort of a 'buffer' of visible map that 'pushes' the screen scrolling.